src/richText.js

```diff
@@ -103,7 +102,18 @@ define([
         }
     });
 
-    CKEDITOR.config.allowedContent = true;
+    CKEDITOR.config.allowedContent = {
+        span: {
+            classes: 'label,label-*',
+            attributes: 'data-output-value,data-value,data-date-format',
+        },
+        i: {
+            classes: 'fcc,fcc-*,fa,fa-*',
+        },
+        output: {
+            attributes: 'value',
+        },
```

Comment:
https://github.com/dimagi/Vellum/blob/7b6c26b3786f5e15dba7722114dc427c6ad185ac/src/markdown.js#L9 
https://confluence.dimagi.com/display/commcarepublic/Text+Formatting

we have a pretty strict whitelist on what we support. 

I also attempted to put html in a form on my phone and it hid the entire text entire text.

```text <b>bold</b>``` would output ```text ``` on a phone